components/eventing-controller/pkg/handlers/nats_test.go

```diff
@@ -340,9 +340,9 @@ func TestIsValidSubscription(t *testing.T) {
 
 }
 
-func checkIsValid(sub *nats.Subscription, t *testing.T) error {
-	return checkValidity(sub, true, t)
-}
+//func checkIsValid(sub *nats.Subscription, t *testing.T) error {
+//	return checkValidity(sub, true, t)
+//}
```

Comment:
If code is not used, can you please delete it?

Reply:
Fixed it